c/busybox-1.22.0/cal_false-unreach-call_true-no-overflow_true-valid-memsafety.c

```diff
@@ -1864,6 +1864,31 @@ int getopt_long(int argc, char * const argv[], const char *optstring,
   return result;
 }
 
+int vasprintf(char **ptr, const char *fmt, va_list ap)
+{
+  (void)*fmt;
+  (void)ap;
+
+  if(__VERIFIER_nondet_int())
+    return -1;
+
+  // limiting the resulting string to at most 42 characters is an
+  // underapproximation obviously
+  *ptr = malloc(42);
```

Comment:
I don't know how well other tools deal with that either. Heck, I am not sure how well SMACK deals with it :). But I don't think that should be our main concern here. Rather, we should focus on creating good models that offer as much coverage as possible while preventing false bugs from happening. So I think that the right thing to do here is to have malloc be invoked with a nondeterministic value. Just my 2 cents.